README.md

```diff
@@ -207,17 +200,14 @@ The example will be transformed to the following output:
 ```
 .../
 ├── images
-│   ├── c_class
-│   │	├── 1.png //Same as 00001.png
-│   ├── q_class
-│   │	├── 2.png //Same as 00002.png
-│   │	├── 3.png //Same as 00001.png
+│   ├── 00001.png
+│   ├── 00002.png
 ├── masks
 │   ├── c_class
-│   │	├── 1.png //Mask merging 00001_c01_c02_q01.png and 00001_c03_c04.png
-│   │	├── 2.png //Same as 00002_q02.png
+│   │	├── 00001.png //Mask merging 00001_c01_c02_q01.png and 00001_c03_c04.png
 │   ├── q_class
-│   │	├── 3.png //Same as 00001_c01_c02_q01.png
+│   │	├── 00001.png //Same as 00001_c01_c02_q01.png
+│   │	├── 00002.png //Same as 00002_q02.png
 ```
 
 ##### Mask merging
```

Comment:
In this section we should remove the `Reverse masks merging example` and add an example for non positive class in binary classification. That would be mask file going fully black

Reply:
Done